### PR TITLE
Add support for Snort VRT subscription rules

### PR DESCRIPTION
**Pull request by shonjir**
Initial support for Snort VRT rules.  Registration or subscription required to obtain valid oinkcode.

### CODE REVIEW

File: security/intrusion-detection-content-snort-vrt/Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		intrusion-detection-content-snort-vrt
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		IDS Snort VRT ruleset (needs registration or subscription)
+PLUGIN_MAINTAINER=	ad@opnsense.org
+PLUGIN_WWW=		https://www.snort.org/downloads#rules
+
```
Comment (fichtner) [inline security/intrusion-detection-content-snort-vrt/Makefile]: missing:

PLUGIN_DEVEL= yes

File: security/intrusion-detection-content-snort-vrt/Makefile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		intrusion-detection-content-snort-vrt
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		IDS Snort VRT ruleset (needs registration or subscription)
+PLUGIN_MAINTAINER=	ad@opnsense.org
```
Comment (fichtner) [inline security/intrusion-detection-content-snort-vrt/Makefile]: I am not sure if we should add ports for other maintainers @AdSchellevis, normally maintainer is submitter

**Comment by fichtner** [comment]
Thanks for this! Assigned to IDS maintainer...

In general, how well does Snort VRT work with Suricata in your experience?

**Comment by shonjir** [comment]
It works pretty well for my production suricata installation - am using it in combination with ET Open and Suricata 4.0 now.  There are a handful of rules that don't load due to slight differences in keyword processing, and of course the binary plugins don't work.

I use enable rules based on the metadata:profile tags (connectivity, balanced, security, max-detect) and modify them for drop mode based on the presence of the "drop" keyword.

**Comment by shonjir** [comment]
Added PLUGIN_DEVEL=yes and squashed the commit

File: security/intrusion-detection-content-snort-vrt/Makefile (hunk 2 above)
Comment (AdSchellevis) [reply security/intrusion-detection-content-snort-vrt/Makefile]: @fichtner @shonjir I don't mind in this case, the template and code behind it is mine, but if @shonjir wants to maintain that's fine by me too, just let me know.

**Comment by fichtner** [comment]
Thanks! We will give this a round of testing as a development package, you can install in OPNsense 17.7.6 with "pkg install os-intrusion-detection-content-snort-vrt-devel"

It's long, but eventually just a click in the GUI ;)

**Pull request merged**